include/date/date.h

```diff
@@ -1718,7 +1718,7 @@ operator<<(std::basic_ostream<CharT, Traits>& os, const year& y)
     detail::save_ostream<CharT, Traits> _(os);
     os.fill('0');
     os.flags(std::ios::dec | std::ios::internal);
-    os.width(4 + (y < year{0}));
+    os.width(4 + int64_t(y < year{0}));
```

Comment:
There's no overflow possible here.  The only two possible expressions are `4 + 0` or `4 + 1`.